Makefile

```diff
@@ -271,12 +258,11 @@ swagger-docs: ## preview the API documentation
 		bfirsh/redoc:1.14.0
 
 .PHONY: buildx
-ifdef USE_BUILDX
 ifeq ($(BUILDX), bundles/buildx)
-buildx: bundles/buildx ## build buildx cli tool
-endif
-endif
-
-bundles/buildx: bundles ## build buildx CLI tool
+buildx: bundles/buildx ## download buildx CLI tool
+bundles/buildx:
 	curl -fsSL https://raw.githubusercontent.com/moby/buildkit/70deac12b5857a1aa4da65e90b262368e2f71500/hack/install-buildx | VERSION="$(BUILDX_VERSION)" BINDIR="$(@D)" bash
 	$@ version
+else
```

Comment:
I'm a bit fuzzy on where we use `bundles/buildx`; is this on the host or inside the container? If it's on the host, can we assume `docker buildx` is pre-installed (requirement). And inside the dev container we can make sure it's installed. (We should have another look at updating the CLI, but possibly keep a second (older) cli around _only_ for the integration-cli tests for now)

Reply:
On the host yes. Looking at what we said in https://github.com/moby/moby/pull/44533#discussion_r1032378391, I go ahead and remove this.